### PR TITLE
Add auto reroute reason to flow history

### PR DESCRIPTION
**Pull request by rozdy**

### CODE REVIEW

File: src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/model/FlowThrottlingData.java [hunk 1]
```diff
@@ -32,20 +32,22 @@
     private Integer priority;
     private Instant timeCreate;
     private Set<IslEndpoint> affectedIsl;
+    private String reason;
 
     @VisibleForTesting
     public FlowThrottlingData(String correlationId, Integer priority) {
-        this(correlationId, priority, Instant.now(), Collections.emptySet());
+        this(correlationId, priority, Instant.now(), Collections.emptySet(), "");
```
Comment (sergii-iakovenko) [inline src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/model/FlowThrottlingData.java]: Why is the empty string used here, not null? Asking as the "reason" field is nullable, so it's possible to set null.

Comment (rozdy) [reply src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/model/FlowThrottlingData.java]: This constructor used for tests only. Changed to null.

File: src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/model/FlowThrottlingData.java [hunk 2]
```diff
@@ -32,20 +32,22 @@
     private Integer priority;
     private Instant timeCreate;
     private Set<IslEndpoint> affectedIsl;
+    private String reason;
 
     @VisibleForTesting
     public FlowThrottlingData(String correlationId, Integer priority) {
-        this(correlationId, priority, Instant.now(), Collections.emptySet());
+        this(correlationId, priority, Instant.now(), Collections.emptySet(), null);
```
Comment (timofei-durakov) [inline src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/model/FlowThrottlingData.java]: does it make sense to remove this constructor at all?

Comment (rozdy) [reply src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/model/FlowThrottlingData.java]: Reworked code to remove this constructor.

**Comment by rozdy** [comment]
Auto reroute reason is fixed by #3295

**Pull request closed without merging**